ext/scheduler/airflow/dag/compiler_test.go

```diff
@@ -28,6 +28,9 @@ var compiledTemplate24 []byte
 //go:embed expected_dag.2.6.py
 var compiledTemplate26 []byte
 
+//go:embed expected_dag.2.9.py
+var compiledTemplate29 []byte
```

Comment:
Please update the documentation in this file `docs/docs/client-guide/defining-scheduler-version.md`

Reply:
just realized we have this. thanks mas!